examples/DancingGoat/appsettings.json

```diff
@@ -16,8 +16,9 @@
   "ConnectionStrings": {
     "CMSConnectionString": ""
   },
+  "WebCrawlerBaseUrl": "http://localhost:23626",
```

Comment:
This was missing, which prevented page crawling for indexing from working 'out of the box'